### DISCUSSION TITLE
User Flows

### DISCUSSION THREAD

**Discussion by ZzRanger** [acmutd/comet_planning#200] [General]
Starting this discussion for user flows of the current iteration of Planner.

**Comment by Afsartech** [comment]
https://www.figma.com/file/VSpB89Q4S1C6HlqRcBOaGM/Nebula-User-Settings---User-Flow?node-id=0%3A1

This is for User Settings

**Comment by jro190002** [comment]
<img width="933" alt="Course and Transfer Credits" src="https://user-images.githubusercontent.com/113809125/193322440-d1f396b2-4477-4815-8c25-067df2b86690.png">
